Google Docs Share/pom.xml

```diff
@@ -2,18 +2,19 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <artifactId>alfresco-googledocs-share</artifactId>
-    <version>${project.parent.version}-${buildnumber}</version>
+    <version>${project.parent.version}</version>
     <packaging>amp</packaging>
     <name>Alfresco / Google Docs Share Module</name>
 
     <properties>
-        <project.build.finalName>${project.build.finalName}-${buildnumber}.amp</project.build.finalName>
+        <alfresco-community.version>6.0.0-SNAPSHOT</alfresco-community.version>
```

Comment:
I assume there is no Share RC yet ? We will need a TODO reminder (in JIRA) to come back and fix this for 3.1.0-RC2.

Reply:
No Share RC is available yet. I've added a TODO.